Documentation/Compilation-Errors.md

```diff
@@ -0,0 +1,134 @@
+# Compilation errors
+
+Swift’s compile errors can be mystifying if you are not intimately familiar with
```

Comment:
"compile errors", or "compilation errors"? "Compilation errors" sounds better to me.